Amazon.Ion.ObjectMapper.Test/IonObjectSerializerTest.cs

```diff
@@ -96,6 +96,21 @@ public void SerializesAndDeserializesCustomPropertyNames()
             Check(TestObjects.fmRadio);
         }
 
+        [TestMethod]
+        public void SerializesAndDeserializesGetterAndSetterMethods()
+        {
+            var serializer = new IonSerializer();
+
+            var stream = serializer.Serialize(TestObjects.honda);
```

Comment:
Won't this lead to a conflict as you have StyleCop changing things like capitalization?

Reply:
Yes it will but I will deal with it when I handle the inevitable merge conflicts later.